app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,52 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+
+    category = benchmark_run.result.keys.map {|key| key}
```

Comment:
`categories` would be a better variable name.

`benchmark_run.result.keys.map {|key| key}` is equivalent to `benchmark_run.result.keys`